--- icu/icu4c/source/Makefile.in
+++ icu/icu4c/source/Makefile.in
***************
*** 38,44 ****
  @TOOLS_TRUE@@TESTS_TRUE@TEST = test
  @SAMPLES_TRUE@SAMPLE = samples
  @TOOLS_TRUE@TOOLS = tools
! @TOOLS_TRUE@DATASUBDIR = data
  
  ## pkgconfig setup. Always have uc and i18n. Others are optional.
  ALL_PKGCONFIG_SUFFIX=uc i18n
--- 38,44 ----
  @TOOLS_TRUE@@TESTS_TRUE@TEST = test
  @SAMPLES_TRUE@SAMPLE = samples
  @TOOLS_TRUE@TOOLS = tools
! @DATA_TRUE@DATASUBDIR = data
  
  ## pkgconfig setup. Always have uc and i18n. Others are optional.
  ALL_PKGCONFIG_SUFFIX=uc i18n
